picard/file.py

```diff
@@ -352,17 +353,27 @@ def _move_additional_files(self, old_filename, new_filename):
         new_path = encode_filename(os.path.dirname(new_filename))
         patterns = encode_filename(config.setting["move_additional_files_pattern"])
         patterns = filter(bool, [p.strip() for p in patterns.split()])
+        try:
+            names = os.listdir(old_path)
+        except os.error:
+            log.debug("Error: {} directory not found".format(old_path))
```

Comment:
Use `log.error()` instead, as debug level messages are only shown when debug mode is on.